ddmrp_report_part_flow_index/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_ddmrp_flow_index_group,ddmrp.flow.index.group,model_ddmrp_flow_index_group,stock.group_stock_manager,1,1,1,1
+access_report_ddmrp_part_plan_flow_index,ddmrp.part.plan.flow.index,model_report_ddmrp_part_plan_flow_index,stock.group_stock_manager,1,1,1,1
```

Comment:
Can you change this two access rights to the group ddmrp.group_ddmrp_manager and add a new one that allows readonly in index group by ddmrp.group_stock_buffer_maintainer